documents/customer.md

```diff
@@ -157,6 +157,248 @@ Razorpay::Customer.fetch(customerId)
+| customer*   | object  | Customer details. [here](https://razorpay.com/docs/payments/payment-gateway/affordability/eligibility-check/#eligibility-check-api)                                                                                            |
+| instruments | object  | Payment instruments on which an eligibility check is required. [here](https://razorpay.com/docs/payments/payment-gateway/affordability/eligibility-check/#eligibility-check-api)                                               |
+
+
+**Response:**
+```json
+{
+  "amount": "500000",
+  "customer": {
+    "id": "KkBhM9EC1Y0HTm",
+    "contact": "+919999999999"
+  },
+  "instruments": [
+    {
+      "method": "emi",
+      "issuer": "HDFC",
+      "type": "debit",
+      "eligibility_req_id": "elig_xxxxxxxxxxxxx",
+      "eligibility": {
+        "status": "eligible"
+      }
+    },
+    {
+      "method": "paylater",
+      "provider": "getsimpl",
+      "eligibility_req_id": "elig_xxxxxxxxxxxxx",
+      "eligibility": {
+        "status": "eligible"
+      }
+    },
+    {
+      "method": "paylater",
+      "provider": "icic",
+      "eligibility_req_id": "elig_xxxxxxxxxxxxx",
+      "eligibility": {
+        "status": "eligible"
+      }
+    },
+    {
+      "method": "cardless_emi",
+      "provider": "walnut369",
+      "eligibility_req_id": "elig_xxxxxxxxxxxxx",
+      "eligibility": {
+        "status": "ineligible",
+        "error": {
+          "code": "GATEWAY_ERROR",
+          "description": "The customer has not been approved by the partner.",
+          "source": "business",
+          "step": "inquiry",
+          "reason": "user_not_approved"
+        }
+      }
+    },
+    {
+      "method": "cardless_emi",
+      "provider": "zestmoney",
+      "eligibility_req_id": "elig_xxxxxxxxxxxxx",
+      "eligibility": {
+        "status": "ineligible",
+        "error": {
+          "code": "GATEWAY_ERROR",
+          "description": "The customer has exhausted their credit limit.",
+          "source": "business",
+          "step": "inquiry",
+          "reason": "credit_limit_exhausted"
+        }
+      }
+    },
+    {
+      "method": "paylater",
+      "provider": "lazypay",
+      "eligibility_req_id": "elig_xxxxxxxxxxxxx",
+      "eligibility": {
+        "status": "ineligible",
+        "error": {
+          "code": "GATEWAY_ERROR",
+          "description": "The order amount is less than the minimum transaction amount.",
+          "source": "business",
+          "step": "inquiry",
+          "reason": "min_amt_required"
+        }
+      }
+    }
+  ]
+}
+```
+
+-------------------------------------------------------------------------------------------------------
+
+### Fetch Eligibility by id
+
+```rb
+eligibilityId = "elig_F1cxDoHWD4fkQt"
+Razorpay::Customer.fetchEligibility(eligibilityId)
```

Comment:
please update the function name here

Reply:
updated

https://github.com/razorpay/razorpay-ruby/blob/981d25ccff2474c3b1cbed16657d6a45606f7d35/documents/customer.md?plain=1#L360